python/ray/exceptions.py

```diff
@@ -196,6 +196,14 @@ def __str__(self):
         return "\n".join(out)
 
 
+class SubmitterNodeDiedError(RayError):
```

Comment:
Submitter node is a bit confusing, how about just "RayletDied"?

Reply:
I was actually retrying to eliminate the confusion of which raylet/node died (the one that executes the task or the one that submits the task).

But I can change to RAYLET_DIED and RayletDiedError.

Reply:
You can maybe say "LocalRayletDied" then.